CHANGELOG.md

```
@@ -12,6 +12,31 @@ to a section with the version name.
 > Use these versions **at your own risk** on v15.
 
 ## Unreleased Changes
+
+## 0.68.5
+* fix Could not find Customer Group: Company
+
+## 0.68.5
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)